suggested deps

maelle opened the pull request:
Fix https://github.com/r-universe-org/docs/issues/40
Fix #64

maelle commented:
A case not using the field: https://github.com/cran/prioritizr/blob/0738fe58f55c812a3023e81398d4294487a8c497/DESCRIPTION#L21

jeroen commented:
I think some people will not like we use rlang for no reason. Let's just use base R  `if(!(require("pkg"))` ...

maelle commented:
@jeroen Good point!

- I added the base R solution but also kept the rlang one as that's a handy one.
- I added another approach that was discussed in the rOpenSci slack.

maelle commented:
@jeroen friendly reminder :smile_cat:
